process/guide.md

```diff
@@ -1259,6 +1259,10 @@ simple.
   `env` environment and that function has the `is_macro` attribute set
   to true. Otherwise, it returns false.
 
+  Feel free to adapt this description. For example, it may be more
+  efficient to return the macro itself instead of `true`, or more
+  expressive to fully replace a call with pattern matching.
+
```

Comment:
This is true of lots of parts of the guide. I would be okay with saying something more generic at the top of the "Make-a-Lisp Process". There isn't anything unique about this particularly part. Something along the lines of "Feel free to follow the guide as literally or as loosely as you like. You are here to learn; wandering off the beaten path may be the way you learn best. However, each step builds on the previous steps, so if you are new to Lisp or new to your implementation language then you may want to stick more closely to the guide your first time through to avoid frustration at later steps."